app/controllers/help_controller.rb

```diff
@@ -1,4 +1,10 @@
 class HelpController < ApplicationController
-	before_filter -> { @css_framework = :bootstrap }
-	layout ->(c) { request.format == :mobile ? "application" : "with_header_with_footer" }
-end
+
+  before_filter -> { @css_framework = :bootstrap }
+  layout -> (c) { request.format == :mobile ? "application" : "with_header_with_footer" }
+
+  def faq
+    @data_chat = AppConfig.chat.enabled ? "enabled" : "disabled"
```

Comment:
Is it a usual use-case in d* ?
It seems a bit over-engineered to use a another just for that, right ?